variables.tf

```diff
@@ -151,6 +151,18 @@ variable "subnetwork" {
 
 # DATABASE
 # --------
+variable "database_host" {
+  default     = null
+  description = "Database host"
```

Comment:
```suggestion
  description = "Optional: If you wish to connect TFE to an existing database, pass the host here."
```

Reply:
Looks good 👍🏼 I'll avoid committing the suggestion so that I can still approve the PR.